content/en/docs/concepts/services-networking/_index.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: "Services, Load Balancing, and Networking"
 weight: 60
+description: >
+  Kubernetes networking addresses four concerns.
+  Containers within a Pod use networking to communicate via loopback.
```

Comment:
Do the four concerns follow? If so, could you use a colon after `... four concerns`?

Reply:
I'm not sure how the rendered page would look. 

>   Kubernetes networking addresses four concerns: Containers within a Pod use networking to communicate via loopback.   Cluster networking provides communication between different Pods. The Service resource lets you expose an application running in Pods to be reachable from outside your cluster. You can also use Services to publish services only for consumption inside your cluster.

?

Reply:
Please shorten description. Something like:
Cluster services and networking expose applications to external services and enable communication between Pods.